api/serializers.py

```diff
@@ -385,3 +357,31 @@ class PlaylistDetailSerializer(PlaylistSerializer):
 
     class Meta(PlaylistSerializer.Meta):
         fields = PlaylistSerializer.Meta.fields + ('channel', 'media')
+
+
+class ProfileSerializer(serializers.Serializer):
+    """
+    The profile of the current user.
+
+    """
+    isAnonymous = serializers.BooleanField(source='user.is_anonymous')
+    username = serializers.CharField(source='user.username', required=False)
+    channels = serializers.SerializerMethodField(
+        help_text="List of channels which the user has edit rights on")
+    displayName = serializers.CharField(source='person.displayName', required=False)
+    avatarImageUrl = serializers.SerializerMethodField()
+
+    def get_channels(self, obj):
+        qs = mpmodels.Channel.objects.all().editable_by_user(obj['user'])
```

Comment:
as before - in the interests of having serialisers just do serialisation, would it be possible to move this onto a method on the model - or something similar?